### DISCUSSION TITLE
Is there a way to enable-shift click to select text between two points?

### DISCUSSION THREAD

**Discussion by alkene5** [kovidgoyal/kitty#8021] [Q&A] [answered]
Is there a way to enable-shift click to select text between two points?

**Comment by kovidgoyal** [comment]
The docs tell you in great detail about everything you can do with the mouse.

**Reply by alkene5** [reply]
This feature does not appear to be in the documentation, Is there any hope of adding this feature?

**Comment by kovidgoyal** [comment]
I have no idea how this feature is different from already existing
functionality. If you want to select between two points hold down shift
and drag the mouse between those two point.

If you want to extend a previous selection with a single click, right
clicking does that.

This is all documented at:
https://sw.kovidgoyal.net/kitty/overview/#mouse-features

and

https://sw.kovidgoyal.net/kitty/conf/#mouse-actions

**Reply by alkene5** [reply]
Sometimes the content that needs to be copied may exceed one screen. At this time, the page needs to be scrolled, and it is inconvenient to drag the mouse.

**Comment by kovidgoyal** [comment, marked as answer]
On Mon, Nov 04, 2024 at 04:27:56PM -0800, Alken E wrote:
> Sometimes the content that needs to be copied may exceed one screen. At this time, the page needs to be scrolled, and it is inconvenient to drag the mouse.


Why? Scrolling is automatic when you drag and in any case if you dont
like it for some reason, right click to extend the selection across
however many screens you want.

**Reply by alkene5** [reply]
" right click to extend the selection" I never knew about this function and how to use it. This function can completely replace the ones I mentioned 😊
